use-shopping-cart/src/index.js

```diff
@@ -14,6 +14,8 @@ import PropTypes from 'prop-types'
 
 export { formatCurrencyString }
 
+export const isClient = typeof window === 'object'
```

Comment:
Why not export the same one from util.js so there's no fragmentation if one changes:

```suggestion
export { formatCurrencyString, isClient } from './util'
```